PIL/aliases.py

```diff
@@ -0,0 +1,12 @@
+from typing import Tuple
+
+# Type aliases; names subject to change
+LURD = Tuple[int, int, int, int]  # left, up(per), right, down = x0, y0, x1, y1
+XY = Tuple[int, int]
+Coord = XY
+Size = XY  # NOTE: All XY aliases will be interchangeable
+Matrix4 = Tuple[float, float, float, float]
+Matrix12 = Tuple[float, float, float, float, float, float, float, float, float, float, float, float]
+Mode = str
+Extrema = Union[Tuple[Any, Any], Tuple[Tuple[Any, Any], ...]]  # Any -> float?
```

Comment:
Pixels are going to either be:

Single int/float, of up to 32 bit width, or tuples of 1 to 4 8 bit ints

I'm not sure if the type system can or should distinguish uint8 vs int32 at the python level.